mods/default/functions.lua

```diff
@@ -363,8 +357,46 @@ minetest.register_abm({
 		-- Remove node
 		minetest.remove_node(pos)
 		minetest.check_for_falling(pos)
-	end
-})
+end
+
+--~ minetest.register_abm({
```

Comment:
delete this entire section - we don't want to keep old code around commented. The git history already keeps a copy of it.